### PR TITLE
Topic edits

### PR DESCRIPTION
**Pull request by philip-menzies**
docs\get_started\plan_deployment\traditional_deployment\roadmaps\rm_creating_site\rm_4site.md docs\get_started\plan_deployment\traditional_deployment\roadmaps\rm_creating_site\rm_setup_content_author\rm_create_simple_theme.md docs\get_started\plan_deployment\traditional_deployment\roadmaps\rm_creating_site\wcm_roadmaps_import.md docs\get_started\plan_deployment\traditional_deployment\roadmaps\rm_install_deployment\rm_standalone_servers\rm_demo.md docs\get_started\plan_deployment\traditional_deployment\roadmaps\rm_install_deployment\rm_standalone_servers\rm_portlet_theme_dev.md docs\get_started\plan_deployment\traditional_deployment\roadmaps\rm_install_deployment\rm_standalone_servers\rm_standalone_parent.md

### CODE REVIEW

File: docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_creating_site/wcm_roadmaps_import.md [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
 
-The Web Content Integrator is a solution for integrating externally managed Web content with HCL Portal. Through the use of standard content syndication feed technologies based on RSS 2.0, the Web Content Integrator provides a loosely-coupled mechanism for transferring published content and metadata to the portal after they have been approved in the source system. When the content and metadata have been transferred to the portal, it is possible to use the built-in content management features of HCL Portal to secure, personalize, and display the content to users.
+The Web Content Integrator \(WCI\) is a solution for integrating externally managed web content with HCL Portal. Through the use of standard content syndication feed technologies based on RSS 2.0, the Web Content Integrator provides a loosely-coupled mechanism for transferring published content and metadata to the portal after they have been approved in the source system. When the content and metadata have been transferred to the portal, it is possible to use the built-in content management features of HCL Portal to secure, personalize, and display the content to users.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_creating_site/wcm_roadmaps_import.md:13]:
```suggestion
The Web Content Integrator (WCI) is a solution for integrating externally managed web content with HCL Portal. Through the use of standard content syndication feed technologies based on RSS 2.0, the Web Content Integrator provides a loosely-coupled mechanism for transferring published content and metadata to the portal after they have been approved in the source system. When the content and metadata have been transferred to the portal, it is possible to use the built-in content management features of HCL Portal to secure, personalize, and display the content to users.
```

File: docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_install_deployment/rm_standalone_servers/rm_portlet_theme_dev.md [hunk 2]
```diff
@@ -37,102 +37,91 @@ Installing HCL Digital Experience involves preparing your operating system, inst
...
 
 2.  Access the Configuration Wizard. Go to http://your_server:10200/hcl/wizard.
 
-    !!!note
-        If working with HCL Digital Experience 8.5 or 9 software level prior to CF18, the wizard address will be: http://your_server:10200/ibm/wizard. After installing CF18, the configuration wizard will automatically be adjusted to http://your_server:10200/hcl/wizard.
+If you are working with HCL Digital Experience 8.5 or 9 at a software level prior to CF18, the wizard address is http://your_server:10200/ibm/wizard. After you install CF18, the configuration wizard is automatically adjusted to http://your_server:10200/hcl/wizard.
 
-    !!!note "Restriction"
-        There is a known issue with Chrome version 45.x and the Configuration Wizard. If you are experiencing difficulties, use a different browser when you access the wizard.
+There is a known issue with Chrome version 45 and later and the Configuration wizard. If you experience difficulties, use a different to access the wizard.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_install_deployment/rm_standalone_servers/rm_portlet_theme_dev.md:98]:
```suggestion
There is a known issue with Chrome version 45 and later and the Configuration wizard. If you experience difficulties, use a different browser to access the wizard.
```

File: docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_install_deployment/rm_standalone_servers/rm_portlet_theme_dev.md [hunk 3]
```diff
@@ -37,102 +37,91 @@ Installing HCL Digital Experience involves preparing your operating system, inst
...
-
-1.  To get the latest updates for the wizard, apply the most recent Combined Cumulative Fix. For more information about applying the latest fix pack, visit [Apply Combined Cumulative Fix](../../../../../../deployment/install/traditional/cf_install/index.md) for more topic information.
+    If you set **Use Administrator IDs stored in your LDAP user registry** to ``yes``, the WebSphere® Application Server and HCL Portal user IDs and passwords are changed to the LDAP user ID and password. If you do not want to change both user IDs and passwords to match the LDAP user ID and password, set this value to ``no``. After you configure your LDAP user registry, you can manually change the user IDs and passwords.
 
-    !!!note
-        Skip this step, if you have the most recent fix pack applied.
+1. Skip this step, if you have the most recent fix pack applied. To get the latest updates for the wizard, apply the most recent CF. For more information about applying the latest fix pack, visit [Apply Combined Cumulative Fix](../../../../../../deployment/install/traditional/cf_install/index.md) for more topic information.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_install_deployment/rm_standalone_servers/rm_portlet_theme_dev.md:92]:
```suggestion
1. Skip this step, if you have the most recent fix pack applied. To get the latest updates for the wizard, apply the most recent CF. For more information about applying the latest fix pack, visit [Apply Combined Cumulative Fix](../../../../../../deployment/install/traditional/cf_install/index.md).
```

**Review by biancaluna — CHANGES_REQUESTED** [review]
Hi @philip-menzies , please see suggested changes. Thanks!

File: docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_creating_site/wcm_roadmaps_import.md (hunk 1 above)
Comment (philip-menzies) [reply docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_creating_site/wcm_roadmaps_import.md:13]: Yes

File: docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_install_deployment/rm_standalone_servers/rm_portlet_theme_dev.md (hunk 2 above)
Comment (philip-menzies) [reply docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_install_deployment/rm_standalone_servers/rm_portlet_theme_dev.md:98]: Yes

File: docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_install_deployment/rm_standalone_servers/rm_portlet_theme_dev.md (hunk 3 above)
Comment (philip-menzies) [reply docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_install_deployment/rm_standalone_servers/rm_portlet_theme_dev.md:92]: Yes

**Review by biancaluna — APPROVED** [review]

**Pull request merged**